### PR TITLE
[HW-3] by den_havanski (4_methods_for_logs_processing)

### PR DESCRIPTION
**Pull request by den-kak-den**

### CODE REVIEW

File: HW03/.idea/modules.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/HW03.iml" filepath="$PROJECT_DIR$/.idea/HW03.iml" />
+    </modules>
+  </component>
+</project>
```
Comment (vladhilko) [inline HW03/.idea/modules.xml:8]: Could you remove these files?

File: HW03/Denis_Havanski/den_havanski_hw_03_t_03.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    i += 1
+  end
+
+  if result.size == 1
+    result[0]
+  elsif result.empty?
+    '0'
+  else
+    result
+  end
```
Comment (vladhilko) [inline HW03/Denis_Havanski/den_havanski_hw_03_t_03.rb:34]: Could you try to rewrite this logic in a more readable way somehow? using `each_cons(2)` for example?

File: HW03/.idea/modules.xml (hunk 1 above)
Comment (den-kak-den) [reply HW03/.idea/modules.xml]: Of course. The files are deleted. Check please

File: HW03/Denis_Havanski/den_havanski_hw_03_t_03.rb (hunk 2 above)
Comment (den-kak-den) [reply HW03/Denis_Havanski/den_havanski_hw_03_t_03.rb]: Done. Thanks for good idea!

**Review by vladhilko — APPROVED** [review]

**Pull request merged**